app/components/swimlanes/swimLanes.tsx

```diff
@@ -128,6 +145,7 @@ const SwimLanes = ({ lanesWithNumItems }) => {
       </Link>
     </Box>
   ));
+  return <>{lanes}</>;
```

Comment:
Any issue with React children?

Reply:
Yes it's mad that it's returning an array